deploy/role.yaml

```diff
@@ -96,6 +96,13 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
There are 3 roles defined in this file

1. role for the namespace the network-operator is deployed in . 
2. role for the namespace the operator deploys its assets (e.g daemosets). this is a fixed namespace: `mlnx-network-operator-resources`
3. cluster role for cluster scoped objects

i this these changes should be under 2.

Reply:
just the latter `mlnx-network-operator-resources`, sorry for not being clear here (reading my own comment after two weeks, i can understand why this wasnt clear :) )